docs/userguides/alertrules.md

```diff
@@ -0,0 +1,122 @@
+# Associate Users to Alert Rules
+
+Once you [create an alert rule in the Code42 console](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Alert_rule_settings_reference), you can use the CLI add and remove users from your existing alert rules.
+
+Use the `alert-rules` commands to manage which users belong to each rule.
```

Comment:
this line feels a bit redundant, maybe just adjust the first sentence above to be `use the CLI 'alert-rules' commands to...`